cmd/util/cmd/rollback-executed-height/cmd/rollback_executed_height.go

```diff
@@ -241,14 +242,17 @@ func removeForBlockID(
 		return fmt.Errorf("could not find result for block %v: %w", blockID, err)
 	}
 
+	chunkIDs := make([]flow.Identifier, 0, len(result.Chunks))
 	for _, chunk := range result.Chunks {
 		chunkID := chunk.ID()
-		// remove chunk data pack
-		err := chunks.BatchRemove(chunkID, chunkBatch)
+		chunkIDs = append(chunkIDs, chunkID)
+	}
+
+	if len(chunkIDs) > 0 {
+		err := chunks.BatchRemove(chunkIDs, writeBatch, chunkBatch)
```

Comment:
BatchRemove now takes two batch, one for batch operation on protocol db, the other on chunk data pack db.